modules/local/create_samplesheet.nf

```diff
@@ -0,0 +1,35 @@
+process CREATE_SAMPLESHEET {
+    tag "$meta.id"
+
+    executor 'local'
+    memory 100.MB
```

Comment:
Fair enough, I did it this way because that's how nf-core/fetchngs does it (e.g., https://github.com/nf-core/fetchngs/blob/master/modules/local/sra_to_samplesheet/main.nf), but I removed that part of the module and it seems to work fine.